libs/application/ui-shell/src/components/FormExternalDataProvider.tsx

```diff
@@ -125,7 +124,7 @@ const FormExternalDataProvider: FC<{
           <Icon icon="download" size="medium" color="blue400" type="outline" />
         </Box>
         <Text variant="h4">
-          Eftirfarandi gögn verða sótt rafrænt með þínu samþykki
+          {subTitle || 'Eftirfarandi gögn verða sótt rafrænt með þínu samþykki'}
```

Comment:
Wondering if we want to have the hardcoded fallback here? And rather just do:
`subtitle && ....`
My worry is when we change the form and this doesn't update that it could be confusing. Just a thought 🤔

Reply:
This is done for 'backwards compatibility' if I change this as you suggested I would need to change all the projects that currently use the `FormExternalDataProvider` 